server/middleware/populatePrisonConfig.ts

```diff
@@ -0,0 +1,18 @@
+import { RequestHandler } from 'express'
+import AppointmentService from '../services/appointmentService'
+import logger from '../../logger'
+
+export default function populatePrisonConfig(appointmentService: AppointmentService): RequestHandler {
+  return async (req, res, next) => {
+    req.middleware ??= {}
+
+    try {
+      req.middleware.prison = await appointmentService.getPrison(res.locals.user.activeCaseLoadId, res.locals.user)
```

Comment:
This was previously being carried out in the now removed `populatedCurrentUser.js`, it is still needed.

Reply:
Maybe seems a bit odd that the prison data is coming from the `appointmentsService` but c'est la vie.